src/i18n/strings/en_EN.json

```diff
@@ -1194,6 +1194,10 @@
     "Today": "Today",
     "Yesterday": "Yesterday",
     "View Source": "View Source",
+    "Encryption enabled": "Encryption enabled",
+    "Messages in this room are end-to-end encrypted. Learn more & verify this user in their user profile.": "Messages in this room are end-to-end encrypted. Learn more & verify this user in their user profile.",
+    "Encryption not enabled": "Encryption not enabled",
+    "The encryption used by this room isn't supported": "The encryption used by this room isn't supported",
```

Comment:
I think a new file for the component is missing? This text is not used anywhere in the PR...?

Reply:
oops, thank you